reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreatePotentialIssuesReportRuleProvider.java

```diff
@@ -0,0 +1,105 @@
+package org.jboss.windup.reporting.rules.generation;
+
+import java.util.Collections;
+
+import org.jboss.windup.config.AbstractRuleProvider;
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.config.metadata.MetadataBuilder;
+import org.jboss.windup.config.operation.iteration.AbstractIterationOperation;
+import org.jboss.windup.config.phase.ReportGenerationPhase;
+import org.jboss.windup.config.query.Query;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.WindupConfigurationModel;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.graph.service.GraphService;
+import org.jboss.windup.reporting.model.ApplicationReportModel;
+import org.jboss.windup.reporting.model.MigrationIssuesReportModel;
+import org.jboss.windup.reporting.model.TaggableModel;
+import org.jboss.windup.reporting.model.TemplateType;
+import org.jboss.windup.reporting.service.ApplicationReportService;
+import org.jboss.windup.reporting.service.ReportService;
+import org.jboss.windup.util.exception.WindupException;
+import org.ocpsoft.rewrite.config.ConditionBuilder;
+import org.ocpsoft.rewrite.config.Configuration;
+import org.ocpsoft.rewrite.config.ConfigurationBuilder;
+import org.ocpsoft.rewrite.context.EvaluationContext;
+
+/**
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+public class CreatePotentialIssuesReportRuleProvider extends AbstractRuleProvider
+{
+    public static final String POTENTIAL_ISSUES = "Potential Issues";
+    public static final String TEMPLATE_CATCHALL_REPORT = "/reports/templates/migration-issues.ftl";
```

Comment:
I think so too, catchall findings are just informative, so you could get the imagination what you need to check/replace and create a script or something for automated replacement.